src/com/javafee/task/bankaccount/BankAccount.java

```diff
@@ -0,0 +1,32 @@
+package src.com.javafee.task.bankaccount;
+
+
+public class BankAccount {
+
+
+    private double balance;
+
+    public double getBalance() {
+        return balance;
+    }
+
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+    void withdraw(double withdraw_anmount) {
```

Comment:
Could u please add the check if the client has enough money on his/her account, please?